Tools/TestWebKitAPI/Tests/WebKitCocoa/LoadAndDecodeImage.mm

```diff
@@ -156,4 +159,46 @@
     Util::run(&done);
 }
 
+TEST(WebKit, GetInformationFromImageData)
+{
+    RetainPtr webView = adoptNS([WKWebView new]);
+    done = false;
+    RetainPtr pngData = [NSData dataWithContentsOfURL:[NSBundle.test_resourcesBundle URLForResource:@"icon" withExtension:@"png"]];
+    [webView _getInformationFromImageData:pngData.get() completionHandler:^(NSString *typeIdentifier, NSArray<NSValue *> *availableSizes, NSError *error) {
+        EXPECT_NULL(error);
+        EXPECT_TRUE([typeIdentifier isEqualToString:UTTypePNG.identifier]);
+        EXPECT_EQ(1u, availableSizes.count);
+        for (NSValue *size in availableSizes) {
+            EXPECT_EQ(215, [size sizeValue].width);
+            EXPECT_EQ(174, [size sizeValue].height);
+        }
```

Comment:
Why do we have to loop when the count is 1?

Reply:
To avoid crash if `availableSizes.count` is not 1; I updated to use `firstObject`